CONTRIBUTING.md

```diff
@@ -24,6 +24,15 @@ git submodule update --init && node scripts/buildocaml.js
 
 `scripts/ninja.js` will generate many `.ninja` build files inside the `jscomp` directory which will be invoked by `./scripts/ninja.js build`.
 
+### Install JS tools
+
+This is required to have all the necessary tools installed, such as `mocha` and
+`istanbul` etc.
+
+```
+npm install
+```
```

Comment:
`istanbul` is not actively used, I will do a clean up on dev dependency, thanks

Reply:
Right, but `mocha` is still in use I assume? So either there is some instruction to install a specific mocha version globally, or we stick to the `npm install` process and check in the resulting `package-lock.json` file?

Reply:
removed all occurrences of istanbul / npm cover

Reply:
mocha is used in testing. "we stick to the npm install process and check in the resulting package-lock.json file?" sounds good to me